cms/home/templates/cms_home/blocks/three_column_text_block.html

```diff
@@ -9,13 +9,13 @@ <h3 class="three-column-text-block-title">{{ value.title }}</h3>
         {% endif %}
         <div class="row equal">
             <div class="col-md-4 col-sm-4">
-                {{ value.col1 }}
+                {% include "cms_home/blocks/teasertext_block.html" with value=value.col1 %}
```

Comment:
Without this, updating wagtail resulted in the templates not being found anymore (on the homepage was just the unstyled StructBlock).
I am not sure this fix is the best idea/the right way to do it, though.